### PR TITLE
Add more supported countries

### PR DESCRIPTION
**Pull request by Primdahl26**
### Is your feature request related to a problem? Please describe.

We need more supported countries. We have the *POWER* to handle more countries. We should do it!

### Describe the solution you'd like

JUST DO IT.. Nah but `navbar.svelte` holds the frontend part, and `config.py` holds the backend part.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### CODE REVIEW

File: frontend/src/components/navbar.svelte [hunk 1]
```diff
@@ -73,7 +96,7 @@
     >
       <option disabled={true}>Choose country</option>
       {#each countries as country}
-        <option value={country.value}>{country.name}</option>
+        <option value={country.value}>{country.icon} {country.name}</option>
       {/each}
     </select>
```
Comment (AndreasPB) [inline frontend/src/components/navbar.svelte:101]: The dropdown needs to be searchable now IMO - Could use the multiselect for this, I'm pretty sure it was a "don't be a multi-select"-option

File: frontend/src/components/navbar.svelte [hunk 2]
```diff
@@ -8,11 +8,34 @@
...
+    { name: "Bulgaria", value: "BG", icon: "🇧🇬" },
+    { name: "Croatia", value: "HR", icon: "🇭🇷" },
+    { name: "Czechia", value: "CZ", icon: "🇨🇿" },
+    { name: "Estonia", value: "EE", icon: "🇪🇪" },
+    { name: "Greece", value: "GR", icon: "🇬🇷" },
+    { name: "Ireland", value: "IE", icon: "🇮🇪" },
+    { name: "Latvia", value: "LV", icon: "🇱🇻" },
+    { name: "Poland", value: "PL", icon: "🇵🇱" },
+    { name: "Romania", value: "RO", icon: "🇷🇴" },
   ]
```
Comment (AndreasPB) [inline frontend/src/components/navbar.svelte:39]: I think it would make a lot more sense if it was in alphabetic order

**Review by AndreasPB — CHANGES_REQUESTED** [review]
Love the flags, but I have a few things

**Review by AndreasPB — CHANGES_REQUESTED** [review]
I guess we need to figure out what to do with the flags since Windows is being... Windows

We might have to have all flags stored as .png or something

File: frontend/src/components/navbar.svelte (hunk 1 above)
Comment (Primdahl26) [reply frontend/src/components/navbar.svelte:101]: Hmmm, not sure if i agree - this would hurt mobile use quite a bit imo

Comment (AndreasPB) [reply frontend/src/components/navbar.svelte:101]: Fair point, we should probably not over-engineer a country selector :D

File: frontend/src/components/navbar.svelte (hunk 2 above)
Comment (Primdahl26) [reply frontend/src/components/navbar.svelte]: Done

**Review by AndreasPB — APPROVED** [review]
I honestly think we should just have this and deal with the Windows "problem".

Added an issue for the .txt stuff #142

**Pull request merged**